docs/DEVELOPER.md

```diff
@@ -84,6 +84,7 @@ You can find more info about debugging [tests with Bazel in the docs.](https://g
 
 - Run: `node tests/legacy-cli/run_e2e.js`
```

Comment:
Can you add a line here that now running `yarn build` needs to be ran prior to `node tests/legacy-cli/run_e2e.js` as the latter will not build the packages.

Reply:
Updated